src/amo/components/SearchForm/styles.scss

```diff
@@ -28,6 +28,13 @@
   white-space: nowrap;
   width: 100%;
 
+  &:focus,
+  &:hover {
+    @include focus();
```

Comment:
Can we split `:focus` and `:hover` so that the  `focus()` mixin is *only* used for focus?

The hover style should be a 1px border using `$teal-70` rather than `$teal-50`.